handlers/api.go

```diff
@@ -1251,6 +1257,11 @@ func getSyncCommitteeSlotsStatistics(validators []uint64, epoch uint64) (types.S
 			syncStats := utils.AddSyncStats(vs[:latestPeriodCount], res, nil)
 			// if latest returned period is the active one, add remaining scheduled slots
 			firstEpochOfPeriod := utils.FirstEpochOfSyncPeriod(syncCommitteeValidators[0].Period)
+			if firstEpochOfPeriod < utils.Config.Chain.ClConfig.AltairForkEpoch {
+				// the first actual sync period starts at the altair fork epoch and might be shorter than all others
+				// https://eth2book.info/capella/annotated-spec/#sync-committee-updates
+				firstEpochOfPeriod = utils.Config.Chain.ClConfig.AltairForkEpoch
+			}
```

Comment:
This should probably go after line 1265, since `lastEpochOfPeriod` is calculated as `firstEpochOfPeriod + EpochsPerSyncCommitteePeriod`, which will not be true if `firstEpochOfPeriod` is set to the altair epoch.

Reply:
Good catch, fixed.